main/src/library/Map.flix

```diff
@@ -562,6 +557,25 @@ namespace Map {
             case _  => foldLeftWithKey((b, k, v) -> if (f(k, v)) b + 1 else b, 0, m)
         }
 
+    ///
+    /// Returns the sum of all keys in the map `m`.
+    ///
+    pub def sumKeys(m: Map[Int32, v]): Int32 =
+        sumWith((k, _) -> k, m)
+
+    ///
+    /// Returns the sum of all values in the map `m`.
+    ///
+    pub def sumValues(m: Map[k, Int32]): Int32 =
+        sumWith((_, v) -> v, m)
+
+    ///
+    /// Returns the sum of all key-value pairs `k => v` in the map `m` according to the function `f`.
+    ///
+    // TODO: Purity polymorphic
```

Comment:
Move the TODO between the signature and the call to foldLeft-- that way it does not become part of the doc.